src/angular/codemods/app-config.ts

```diff
@@ -0,0 +1,232 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+import type { ArrayExpression, Identifier, ObjectProperty } from '@babel/types';
+
+// @ts-expect-error - magicast is ESM and TS complains about that. It works though
+import type { ProxifiedModule } from 'magicast';
+import { gte, type SemVer } from 'semver';
+import * as recast from 'recast';
+
+export function updateAppConfigMod(
+  originalAppConfigMod: ProxifiedModule<any>,
+  angularVersion: SemVer,
+  isTracingEnabled: boolean,
+): ProxifiedModule<any> {
+  const isAboveAngularV19 = gte(angularVersion, '19.0.0');
+
+  addImports(originalAppConfigMod, isAboveAngularV19, isTracingEnabled);
+  addProviders(originalAppConfigMod, isAboveAngularV19, isTracingEnabled);
+
+  return originalAppConfigMod;
+}
+
+function addSentryImport(originalAppConfigMod: ProxifiedModule<any>): void {
+  const imports = originalAppConfigMod.imports;
+  const hasSentryImport = imports.$items.some(
+    (item) =>
+      item.from === '@sentry/angular' &&
+      item.imported === '*' &&
+      item.local === 'Sentry',
+  );
+
+  if (!hasSentryImport) {
+    imports.$add({
+      from: '@sentry/angular',
+      imported: '*',
+      local: 'Sentry',
+    });
+  }
+}
+
+function addErrorHandlerImport(
+  originalAppConfigMod: ProxifiedModule<any>,
+): void {
+  const imports = originalAppConfigMod.imports;
+  const hasErrorHandler = imports.$items.some(
+    (item) => item.local === 'ErrorHandler',
+  );
+
+  if (!hasErrorHandler) {
+    imports.$add({
+      from: '@angular/core',
+      imported: 'ErrorHandler',
+      local: 'ErrorHandler',
+    });
+  }
```

Comment:
m: If they already have their own ErrorHandler, users need to combine theirs with ours. Let's print a link to our docs on how to do that. https://docs.sentry.io/platforms/javascript/guides/angular/features/error-handler/#multiple-error-handlers